docs/Installation.md

```diff
@@ -277,18 +277,18 @@ sudo install -v -m 755 ./zm-backend.sh /etc/init.d/
 Check the [declaration of prerequisites] to make sure your preferred combination
 of operating system version and database engine version is supported.
 
-#### 4.2.1 Instructions for MySQL (Debian)
+#### 4.2.1 Instructions for MariaDB (Debian)
 
 Configure Zonemaster::Backend to use the correct database engine:
 
 ```sh
-sudo sed -i '/\bengine\b/ s/=.*/=MySQL/' /etc/zonemaster/backend_config.ini
+sudo sed -i '/\bengine\b/ s/=.*/= MySQL/' /etc/zonemaster/backend_config.ini

```

Comment:
The headline says MariaDB but the configuration file says MySQL. To avoid that the user starts wondering if that is a typo I think it should be stated, that "Yes, in the configuration file it should say 'MySQL'".

Reply:
I think we can safely wait with doing that until we get the first report of a confused user.

Reply:
A comment does not hurt, does it?

Reply:
In this case I don't think it helps anyone.

Reply:
Since it does not hurt, please add it.

Reply:
I do think you should add MariaDB is a fork of MySQL. There are many who does not know it.

Reply:
This information fits better in the [configuration documentation](
https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#db-section) than here. I've updated #520 so we don't forget to include this clarification.

Reply:
Then we should have a reference to that configuration document.